README.md

```diff
@@ -1,2 +1,130 @@
-# Hello-world
-This s my first repository in Microverse
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
+
+> This is my first Microverse Project. It contains every first step I have to take as a developer.
+
+**[Hello Microverse Project]** is a website that is used to test for different linter errors that could occur when building a HTML and CSS project.
+
+## 🛠 Built With <a name="HTML and CSS"></a>
+
+### Tech Stack <a name="Front end"></a>
+
+> This is a desk-top browser website.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
- [ ] I request to consider removing the guiding statements. These are purposely to guide you what to include under that section. Therefore to make your readme professional, you should always consider removing them ☺️.